container/fs/fs_mocks.go

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
Here also test type should be defined - to be clear that these mocks are used in small tests - see https://github.com/intelsdi-x/snap/blob/master/control/fixtures/fixtures.go#L1